README.md

```diff
@@ -114,6 +114,10 @@ $ kn event send \
 
 ## Install
 
+There are multiple ways to install the `kn-plugin-event`.
+
+### Install using a binary
+
```

Comment:
Adding a link to knative.dev docs? https://knative.dev/docs/client/kn-plugins/#manually-install-a-plugin

Reply:
Well of course I've forgot my idea here. I'll check the whole thing once again. In general I'd point to the content on knative.dev when existing already.